### PR TITLE
Addition of Name Spacing to PAA Ext

### PR DESCRIPTION
**Pull request by hillslatt**
**New**

- Section: Non-ORTB Signaling 
- Object: InterestGroupAuctionBuyerSignals
- Attributes: 
- - InterestGroupAuctionSupport.pbsmap
- - InterestGroupAuctionBuyer.pbsmap
- Additional Implementation Guidance for name spacing

### CODE REVIEW

File: extensions/community_extensions/Protected Audience Support.md [hunk 1]
```diff
@@ -31,6 +31,15 @@
     1 = yes, the buyer is allowed
     </td>
   </tr>
+<tr>
+    <td><code>pbsmap</code></td>
+    <td>integer;<br>default 0</td>
+    <td>Indicates whether the seller can provide <code>igb.pbs</code> in <code>Object: InterestGroupAuctionBuyerSignals</code></br>
+    where <br>
+	0 = no; igb.pbs value is passthrough (DEFAULT)<br>
```
Comment (patmmccann) [inline extensions/community_extensions/Protected Audience Support.md:39]: We should just choose the best place, not provide two different places to put the thing. We should only do 1, yes, below

File: extensions/community_extensions/Protected Audience Support.md [hunk 2]
```diff
@@ -310,3 +351,65 @@ Following extends the Ad Served On Win Notice example to demonstrate a seller pl
...
+#### Namespacing in perBuyerSignals
+Early PAAPI auction provisioning practice was for buyers to supply their <code>perBuyerSignals</code> to seller partners as a passthrough value in <code>BidResponse.ext.igbid[].igbuyer[].buyerdata</code> or some other means. The seller places this in the buyer’s key in <code>auctionConfig.perBuyerSignals</code> and Protected Audience then provides the value as the <code>perBuyerSignals</code> parameter to <code>generateBid</code>. This does not afford a clean way for the seller to provide signals to a specific buyer, such as Deals.
+
+To accommodate this, the community extensions support a more open <code>perBuyerSignals</code> handling. In an OpenRTB BidRequest sellers can signal their ability to provide and in the BidResponse buyers may opt in to receive an <code>InterestGroupAuctionBuyerSignals</code> object in the interest group auction. 
+
+If buyer "https://dsp.example"’s igb.pbsmap is missing or 0, then the seller/SSP provides perBuyerSignals as:
+
+```
+"perBuyerSignals": {
+"https://dsp.example": ... // BidResponse.ext.igi[].igb[].pbs
```
Comment (rdgordon-index) [inline extensions/community_extensions/Protected Audience Support.md:389]:
```suggestion
   "https://dsp.example": ... // BidResponse.ext.igi[].igb[].pbs
```

File: extensions/community_extensions/Protected Audience Support.md [hunk 3]
```diff
@@ -310,3 +351,65 @@ Following extends the Ad Served On Win Notice example to demonstrate a seller pl
...
+}
+```
+
+If the buyer opts into perBuyerSignals namespacing by returning pbsmap = 1, then the seller/SSP provides perBuyerSignalls as:
+
+```jsonc
+"perBuyerSignals": {
+"https://dsp.example": {
```
Comment (rdgordon-index) [inline extensions/community_extensions/Protected Audience Support.md:397]:
```suggestion
   "https://dsp.example": {
```

**Review by rdgordon-index — COMMENTED** [review]
whitespace changes only

File: extensions/community_extensions/Protected Audience Support.md [hunk 4]
```diff
@@ -310,3 +351,65 @@ Following extends the Ad Served On Win Notice example to demonstrate a seller pl
...
+
+If buyer "https://dsp.example"’s igb.pbsmap is missing or 0, then the seller/SSP provides perBuyerSignals as:
+
+```
+"perBuyerSignals": {
+"https://dsp.example": ... // BidResponse.ext.igi[].igb[].pbs
+}
+```
+
+If the buyer opts into perBuyerSignals namespacing by returning pbsmap = 1, then the seller/SSP provides perBuyerSignalls as:
```
Comment (patmmccann) [inline extensions/community_extensions/Protected Audience Support.md:393]: we should delete this and always namespace

**Review by dmdabbs — CHANGES_REQUESTED** [review]
>BidResponse.ext.igi: This extension to "Object: Bid" 

This is an extension to BidResponse not object Bid. One can no bid and still pass intent to participate in a (potential) PA auciton.

**Pull request merged**